Distance_3/include/CGAL/Distance_3/internal/squared_distance_utils_3.h

```diff
@@ -112,7 +112,7 @@ wdot(const typename K::Point_3 &p,
 }
 
 
-static double diff_of_products(const double a, const double b, const double c, const double d)
+static double diff_of_products(const double& a, const double& b, const double& c, const double& d)
```

Comment:
If I remember correctly there is no gain for a ` double`  making it ` const&`

Reply:
even a penalty sometimes